unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Scripts/SectionsModule/SettingsSectionModel.cs

```diff
@@ -0,0 +1,40 @@
+using DCL.SettingsPanelHUD.Widgets;
+using ReorderableList;
+using UnityEngine;
+
+namespace DCL.SettingsPanelHUD.Sections
+{
+    [System.Serializable]
+    public class SettingsWidgetList : ReorderableArray<SettingsWidgetModel>
+    {
+    }
+
+    [CreateAssetMenu(menuName = "Settings/Configuration/Section", fileName = "SectionConfiguration")]
+    public class SettingsSectionModel : ScriptableObject
```

Comment:
Adding some XML documentation to these classes would be very welcome!

Reply:
Yes, I have still pending to create all the XML documentation for all these classes. I will do it in this PR: https://github.com/decentraland/explorer/pull/1784

Reply:
Done.